### PR TITLE
Add 404 error page

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-theme-bundle/pull/8
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/113
https://github.com/open-orchestra/open-orchestra-docs/pull/70

### CODE REVIEW

File: en/developer_guide/error_pages.rst [hunk 1]
```diff
@@ -71,5 +71,13 @@ to validate the maintenance rewrite and to remove it to put your site on back.
 
 Error 404 - Page not found
 --------------------------
+The 404 error page requires less efforts than the 503 version to be used. Once a 404 node is contributed
+and published, it immediatly replaces the standard 404 error message from Symfony.
```
Comment (alavieille) [inline en/developer_guide/error_pages.rst]: immediately

**Comment by alavieille** [comment]
this page is not in the summary. can you add it ? and in the fixture of open-orchestra-dot-com.

**Comment by ngilain** [comment]
@itkg-alavieille : update done

**Comment by nicolasThal** [comment]
:+1: after a little squash :)

**Comment by alavieille** [comment]
:+1:

**Pull request merged**